uefi/README.md

```diff
@@ -33,23 +33,6 @@ This crate's features are described in [`src/lib.rs`].
 
 [`src/lib.rs`]: src/lib.rs
 
-## User Documentation
```

Comment:
I'm going to add a small replacement in the next PR. But I want to keep this PR small